config/locales/html_fragments/en.yml

```diff
@@ -0,0 +1,211 @@
+en:
+  pages:
```

Comment:
I think this is a great idea but wonder whether we should have a file per page in the `locals/html_fragments/`. It may be worth renaming it to `locals/pages/data_requirements/en.yml` and eventually `locals/pages/data_requirements/cy.yml`. If we don't have many static pages then we could just put them all in `locals/pages/`. `pages` `content` etc i'm not sure which makes sense

Reply:
One downside to this is the markup will be duplicated for each locale, unless we keep the markup on the page and move the content into keys using the above structure

Reply:
Agreed, but since we will probably have a max of two locales I think it will be easier to update them if we dont add all the different bits of content as individual keys